packages/fxa-auth-server/package.json

```diff
@@ -9,14 +9,15 @@
     "test": "test"
   },
   "scripts": {
-    "build": "yarn emails-scss && tsc --build && cp -R config public lib scripts dist/fxa-auth-server",
+    "build": "yarn merge:ftl && yarn emails-scss && tsc --build && cp -R config public lib scripts dist/fxa-auth-server",
+    "clean": "git clean -xfd",
```

Comment:
This removes all local untracked files.  Seems dangerous.  (But I also cannot find where it is used.)

Reply:
That's a good point. I had a fairly lengthy set of `rm -rf` commands, and then switched to this. It's must be invoked by the developer currently. But I can see how that could be very bad. I'll change it to just the remove the git ignored files. Thanks for the call out 👍